specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2019-01-21/examples/KustoDataConnectionValidation.json

```diff
@@ -7,10 +7,12 @@
     "api-version": "2019-01-21",
     "dataConnectionName": "DataConnections8",
     "parameters": {
-      "kind": "EventHub",
       "properties": {
```

Comment:
I don't believe you want this 'properties'
It should be:
"parameters": { 
"kind": "EventHub",
"properties": {
          "eventHubResourceId": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.EventHub/namespaces/eventhubTestns1/eventhubs/eventhubTest1",
          "consumerGroup": "testConsumerGroup1"
        }
}

Reply:
I missed a property there. Now it should make sense.